uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_leave-calendar.scss

```diff
@@ -5,48 +5,33 @@ $chr-leave-calendar-item-requested-border-height: 0.5em;
 $chr-leave-calendar-legend-size:                  20px;
 
 .chr_leave-calendar {
+  margin-bottom: 0;
 
   @media (min-width: $screen-lg-min) {
     table-layout: fixed;
   }
 }
 
-.chr_leave-calendar__month-header {
-  background: $panel-default-heading-bg;
-}
 
-.chr_leave-calendar__days {
+.chr_leave-calendar__day {
+  font-size: $chr-leave-calendar-cell-font-size !important;;
+  padding: 10px !important;
+  text-align: center !important;
 
-  > th {
-    font-size: $chr-leave-calendar-cell-font-size !important;;
-    padding: 10px !important;
-    text-align: center !important;
-
-    @media (min-width: $screen-lg-min) {
-      padding: 10px 0 !important;
-    }
+  @media (min-width: $screen-lg-min) {
+    padding: 10px 0 !important;
   }
 }
 
-.chr_leave-calendar__items {
-
-  > td {
-    overflow: hidden;
-    height: $chr-leave-calendar-cell-height;
-    padding: 0 !important;
-  }
+.chr_leave-calendar__day--disabled {
```

Comment:
I think we have .chr_disabled{opacity:0.5;pointer-events:none} - will that work?